main.go

```diff
@@ -104,6 +114,26 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "CcRuntime")
 		os.Exit(1)
 	}
+
+	if err = (&peerpodconfigcontrollers.PeerPodConfigReconciler{
```

Comment:
Only enable these reconcilers through a commandline arg so that all users don't get them